src/components/GuardianCard.story.tsx

```diff
@@ -0,0 +1,15 @@
+import { Meta, Story } from '@storybook/react';
+
+import { GuardianCard as GuardianCardComponent } from './GuardianCard';
+
+export default {
+  title: 'App/GuardianCard',
+  component: GuardianCardComponent,
+} as Meta;
+
+export const GuardianCard: Story = () => {
+  return <GuardianCardComponent 
```

Comment:
hah can you format this file? i've included a `.vscode/settings.json` in this repo that should set `prettier` to run automatically on save. u might need to install the extension for vscode?

Reply:
Yeah will take a look